packages/web-extension/test/walletManager/WalletRepository.test.ts

```diff
@@ -344,6 +424,25 @@ describe('WalletRepository', () => {
       ]);
     });
 
+    it('does not remove 1852 account when removing 1854 account', async () => {
+      const newAccount = createAccount(0, 1, KeyPurpose.MULTI_SIG);
```

Comment:
Hardcoding this is brittle, it's very important for this test to use the same accountIndex. Same for `does not update 1852 account metadata when updating 1854 account`
```suggestion
      const newAccount = createAccount(0, storedAccount.accountIndex, KeyPurpose.MULTI_SIG);
```

Reply:
Fixed on 19c238b

Reply:
Because the purpose of the test is to assert that it works well when there is an 1854 and 1852 account with the same `accountIndex`. If that's not clear when reading the test name, please rename it.